ecr_api.py

```diff
@@ -94,16 +94,12 @@ def __init__(self, app):
         self.password = ''
         #self.authenticated = False
 
-    def __call__(self, environ, start_response):
+    def __call__(self, environ, start_response): # pragma: no cover
```

Comment:
This is fine for now.  But in the future I would like to see our authentication code tested using `mock` to simulate calls to `requests.post` (in success and failure conditions)